README.md

```diff
@@ -118,9 +118,9 @@ git commit -m "Initialize the styleguide."
 
   This is a task to deploy the static styleguide to GitHub pages.
 
-  Butler will build a Sculpin production artifact to `styleguide/output_prod` and deploy the production artifact to `gh-pages` branch of the repo defined in the `conf/butler.defaults.js`. Each commit for this process will default to the message: "Updated with Butler - [timestamp]".
+  Butler will build a Sculpin production artifact to and deploy the production artifact to `gh-pages` branch of the repo defined in the `conf/butler.defaults.js`. 
 
-  You may want to create a `sculpin_site_prod.yml` to define the site URL once deployed. You can find out more information about environment aware configuration for Sculpin [here](https://sculpin.io/documentation/configuration/).
+  You may want to create a `sculpin_site_prod.yml` or a `config_prod.yml` (for Spress) to define the site URL once deployed. You can find out more information about environment aware configuration for Sculpin [here](https://sculpin.io/documentation/configuration/) and the configuration for Spress [here](http://spress.yosymfony.com/docs/configuration/).
```

Comment:
@MichellanneLi that looks great! Can you make a PR against this one for the updates to the language?